src/it/resources/data/typicalDataBundle.json

```diff
@@ -483,6 +495,90 @@
         "questionType": "TEXT",
         "answer": "Student 2 self feedback."
       }
+    },
+    "response1ForQ2": {
+      "id": "00000000-0000-4000-8000-000000000903",
+      "feedbackQuestion": {
+        "id": "00000000-0000-4000-8000-000000000802",
+        "feedbackSession": {
+          "id": "00000000-0000-4000-8000-000000000701"
+        },
+        "questionDetails": {
+          "recommendedLength": 0,
+          "questionType": "TEXT",
+          "questionText": "Rate 1 other student's product"
+        },
+        "description": "This is a text question.",
+        "questionNumber": 2,
+        "giverType": "STUDENTS",
+        "recipientType": "STUDENTS_EXCLUDING_SELF",
+        "numOfEntitiesToGiveFeedbackTo": 1,
+        "showResponsesTo": [
+          "INSTRUCTORS",
+          "RECEIVER"
+        ],
+        "showGiverNameTo": [
+          "INSTRUCTORS"
+        ],
+        "showRecipientNameTo": [
+          "INSTRUCTORS",
+          "RECEIVER"
+        ]
```

Comment:
@cedricongjh, @hhdqirui Any reason why responses need the full question instead of just the id?

Reply:
This is because `JsonUtils` has a custom converter for `feedbackQuestion` (I think this is due to it being single table, have to check with @hhdqirui for this), and when deserialising a `feedbackResponse` I discovered that when it was deserialising the `feedbackResponse's` `feedbackQuestion`, it was passing it into the `JsonUtils` feedbackQuestion converter, with only the id field. This results in an error being thrown as it is not a valid feedbackQuestion, and thus the full question was added in.

Reply:
@hhdqirui, @cedricongjh, please help to modify the converter to support this.